src/seeders/slots/SlotSeeder.js

```diff
@@ -12,7 +12,7 @@ class SlotSeeder {
 
   static stubOwningFct(slot) {
     // eslint-disable-next-line
-    slot.functionalityId = uuidv4()
+    slot.functionality = { id: uuidv4() }
```

Comment:
shouldn't this generate a Functionality instead on the right side of the equation?

Reply:
updated!